packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -29,39 +29,30 @@ export const useGetHistoryChats = (): UseGetHistoryChatsResult => {
 	const [ conversations, setConversations ] = useState< Conversations >( [] );
 	const [ supportInteractions, setSupportInteractions ] = useState< SupportInteraction[] >( [] );
 
-	const { data: supportInteractionsOpen, isLoading: isLoadingOpenInteractions } =
+	const { data: openSupportInteraction, isLoading: isLoadingOpenInteractions } =
 		useGetSupportInteractions( 'zendesk', 10, 'open' );
-	const { data: supportInteractionsResolved, isLoading: isLoadingResolvedInteractions } =
-		useGetSupportInteractions( 'zendesk', 100, 'resolved' );
-	const { data: supportInteractionsSolved, isLoading: isLoadingSolvedInteractions } =
-		useGetSupportInteractions( 'zendesk', 100, 'solved' );
-	const { data: supportInteractionsClosed, isLoading: isLoadingClosedInteractions } =
-		useGetSupportInteractions( 'zendesk', 100, 'closed' );
+	const { data: otherSupportInteractions, isLoading: isLoadingOtherSupportInteractions } =
+		useGetSupportInteractions( 'zendesk', 100, [ 'resolved', 'solved', 'closed' ] );
```

Comment:
I grouped those requests thanks to the feature introduced in https://github.com/Automattic/wp-calypso/pull/103072.